Compiler/Template/CodegenCppInit.tpl

```diff
@@ -138,7 +138,7 @@ template scalarVariableAttributeXML(SimVar simVar, SimCode simCode, String index
       let causalityAtt = CodegenFMUCommon.getCausality(causality)
       let variability = getVariablity(varKind)
       let description = if comment then 'description="<%Util.escapeModelicaStringToXmlString(comment)%>"'
-      let additionalAttributes = if generateFMUModelDescription then '' else 'isProtected="<%isProtected%>" isDiscrete="<%isDiscrete%>" isValueChangeable="<%isValueChangeable%>"'
+      let additionalAttributes = if generateFMUModelDescription then '' else 'isProtected="<%isProtected%>" hideResult="<%hideResult%>" isDiscrete="<%isDiscrete%>" isValueChangeable="<%isValueChangeable%>"'
```

Comment:
@ptaeuber @rfranke Do we need both values `hideResult` and `isValueChangeable`?
Maybe it would be enough to make `isValueChangeable` dependent on the HideResult-annotation:
```
isValueChangeable = ((not (BackendVariable.hasVarEvaluateAnnotationOrFinal(dlowVar) or hideResult)
                    and (BackendVariable.varHasConstantBindExp(dlowVar) or not BackendVariable.varHasBindExp(dlowVar))))
                    and isFixed;
```